app/vue/src/client/preview/index.js

```diff
@@ -46,13 +46,18 @@ function decorateStory(getStory, decorators) {
 
       const decoratedStory = decorator((p = {}) => {
         story = decorated(
-          Object.assign(
-            context,
-            p,
-            { parameters: Object.assign(context.parameters || {}, p.parameters) },
-            { options: Object.assign(context.options || {}, p.options) }
```

Comment:
What happens with `options`?

Reply:
Options should never have been there in the first place AFAIK, so we deleted it. Technically a breaking change, but I think this was a bug & all code that I know about would use `parameters.options` which is accounted for in this change.